Model/Classes/User.cs

```diff
@@ -0,0 +1,51 @@
+﻿using AirportTicketBooking.Utils;
+
+namespace AirportTicketBooking.Model.Classes;
+public class User
+{
+    private static long _id = 0;
+    private List<Ticket> tickets = [];
+
+    public long Id { get; init; } = ++_id;
+    public required string Name { get; set; }
+    public required string Email { get; set; }
+    public required string Password { get; set; }
+
+    public bool BookTicket(Ticket ticket)
+    {       
+        tickets.Add(ticket);
```

Comment:
Should we have validation before adding the ticket? to avoid duplicated tickets for example